helm/cas-airflow/values.yaml

```diff
@@ -108,14 +108,113 @@ airflow:
       limits:
         cpu: 500m
         memory: 2Gi
-    defaultUser:
-      enabled: true
-      role: Admin
-      username: cas-airflow-admin
-      email: admin@example.com
-      firstName: admin
-      lastName: user
-      password: "$(DEFAULT_USER_PASS)"
+    # defaultUser:
+    #   enabled: true
+    #   role: Admin
+    #   username: cas-airflow-admin
+    #   email: admin@example.com
+    #   firstName: admin
+    #   lastName: user
+    #   password: "$(DEFAULT_USER_PASS)"
+    webserverConfig: |-
```

Comment:
it might be easier to put all this in a *.tpl file...or even a .py file that is printed out here by a template function

Reply:
We can't use helms templating in values files, so I've moved it to a .py file, and instead of including `webserverConfig` in `values.yaml`, included a `--set-file` for `airflow.webserver.webserverConfig` in the deploy script. Without being able to call `tpl` in the chart (since it's not our chart) this is the only way to do this that I know of. 

If there's a better way to approach this please let me know!

(As of right now it hasn't been tested. Just waiting for the green light to be able to mess with airflow-dev)

Reply:
Either way, it looks far cleaner now. Good call out!